src/test/java/io/asyncer/r2dbc/mysql/ConnectionIntegrationTest.java

```diff
@@ -139,6 +139,31 @@ void transactionDefinitionIsolationLevel() {
             }));
     }
 
+    @Test
```

Comment:
I'm looking to validate that `connection.getTransactionIsolationLevel()` correctly returns the current transaction isolation level. I believe the following test cases should effectively demonstrate this.  WDYT?

```
    @Test
    void setTransactionLevelNotInTransaction() {
        complete(connection ->
                // check initial session isolation level
                         Mono.just(connection.getTransactionIsolationLevel())
                             .doOnSuccess(it -> assertThat(it).isEqualTo(REPEATABLE_READ))
                             .then(connection.beginTransaction())
                             .doOnSuccess(ignored -> assertThat(connection.isInTransaction()).isTrue())
                             .then(Mono.just(connection.getTransactionIsolationLevel()))
                             .doOnSuccess(it -> assertThat(it).isEqualTo(REPEATABLE_READ))
                             .thenMany(Flux.merge(
                                     connection.rollbackTransaction(),
                                     // set transaction isolation level in no transaction
                                     connection.setTransactionIsolationLevel(READ_COMMITTED)
                             ))
                             // check that session isolation level is changed
                             .then(Mono.just(connection.getTransactionIsolationLevel()))
                             .doOnSuccess(it -> assertThat(it).isEqualTo(READ_COMMITTED))
                            .then(connection.beginTransaction())
                            .doOnSuccess(ignored -> assertThat(connection.isInTransaction()).isTrue())
                             // check transaction isolation level applies to subsequent transactions
                            .then(Mono.just(connection.getTransactionIsolationLevel()))
                            .doOnSuccess(it -> assertThat(it).isEqualTo(READ_COMMITTED))
        );
    }

    @Test
    void setTransactionLevelInTransaction() {
        complete(connection ->
                // check initial session transaction isolation level
                         Mono.just(connection.getTransactionIsolationLevel())
                             .doOnSuccess(it -> assertThat(it).isEqualTo(REPEATABLE_READ))
                             .thenMany(Flux.merge(
                                     connection.beginTransaction(),
                                     // set transaction isolation level in transaction
                                     connection.setTransactionIsolationLevel(READ_COMMITTED)
                             ))
                             // check that current transaction isolation level is not changed
                             .then(Mono.just(connection.getTransactionIsolationLevel()))
                             .doOnSuccess(it -> assertThat(it).isNotEqualTo(READ_COMMITTED))
                             .then(connection.rollbackTransaction())
                             .doOnSuccess(ignored -> assertThat(connection.isInTransaction()).isFalse())
                             // check that session isolation level is changed after rollback
                             .then(Mono.just(connection.getTransactionIsolationLevel()))
                             .doOnSuccess(it -> assertThat(it).isEqualTo(READ_COMMITTED))
                             // check transaction isolation level applies to subsequent transactions
                             .then(connection.beginTransaction())
                             .doOnSuccess(ignored -> assertThat(connection.isInTransaction()).isTrue())
        );
    }
 ```